data/categories.txt

```diff
@@ -6073,7 +6080,7 @@ da:WiFi|Wi-Fi
 nl:WiFi|Wi-Fi
 fi:WiFi|Wi-Fi
 fr:WiFi|Wi-Fi
-de:WLAN|WiFi|Wi-Fi
+de:WiFi|Wi-Fi|WLAN
```

Comment:
nit: the order doesn't matter here

Reply:
Alright, in that case, I'll revert this change.